### PR TITLE
fix(HorizontalRule): adds dotted horizontal rule style

### PR DESCRIPTION
**Pull request by pmsorhaindo**
https://5a5f29c9df9953478cb1ba35--librarian-chimpanzee-34483.netlify.com/docs/components/horizontal-rule?state=%7B%22HorizontalRule%22%3A%7B%22props%22%3A%7B%22lineStyle%22%3A%22dotted%22%7D%7D%7D

### CODE REVIEW

File: src/components/horizontal-rule/HorizontalRule.js [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+
   render() {
-    const { ...rest } = this.props;
+    const { style, ...rest } = this.props;
 
     return (
-      <Base space="x0" { ...rest } Component="hr" className="ax-horizontal-rule" />
+      <Base
+          space="x0"
+          style={ { borderStyle: style } }
```
Comment (HHogg) [inline src/components/horizontal-rule/HorizontalRule.js]: I'm not overly worries about blocking passed in inline styles, but it's not something that has to be added inline and could be done through a class. 

Inline styles is generally best used for dynamic values, like widths and heights.

**Comment by thatguynamedandy** [comment]
I think the above would kind of block inline styles because the prop is called `style`.

**Comment by HHogg** [comment]
@thatguynamedandy True, @pmsorhaindo what do you think about going with `borderStyle`?

**Comment by pmsorhaindo** [comment]
I went with lineStyle, but will change to borderStyle.

**Pull request merged**